src/pages/FocusCompareEqualsPage.tsx

```diff
@@ -118,7 +163,7 @@ export const FocusCompareEqualsPage = () => {
       </div>
       {/* Main content */}
       <div>
-        <PackedGrid maxColumns={2}>
+        <PackedGrid maxColumns={3}>
```

Comment:
Why are you changing the `maxColumn`?

Reply:
Hm, I don't really remember doing it :) must be a mistake